packages/apps/tools/src/services/faucet/fund-create-new.ts

```diff
@@ -23,7 +24,7 @@ export const fundCreateNewAccount = async (
   keys: string[],
   chainId: ChainwebChainId,
   amount = 100,
-  pred = 'keys-all',
+  pred: TPredicate,
```

Comment:
The default value should still be present?

Reply:
I don't think so, because Pact will use `keys-all` by default if no predicate is passed.

Reply:
I don't know about pact but its better to have the default value to avoid any unexpected behaviour later